apps/emqx/src/emqx_persistent_session_ds.hrl

```diff
@@ -41,6 +45,8 @@
     ref :: _StreamRef,
     stream :: emqx_ds:stream(),
     rank :: emqx_ds:stream_rank(),
+    topic_filter :: emqx_types:topic(),
+    start_time :: emqx_ds:time(),
```

Comment:
Again, this needs special attention. I followed the same reasoning as [here](https://github.com/emqx/emqx/pull/12029/#discussion_r1405695081), but please shout loudly if it's not acceptable anymore.